_conferences/devfest-mn-2016.md

```diff
@@ -0,0 +1,12 @@
+---
+name: "Devfest MN"
```

Comment:
The other 3 on the list use "GDG DevFest" for their name, relying on location to differentiate.